life_expectancy/loading_saving.py

```diff
@@ -0,0 +1,25 @@
+from pathlib import Path
+
+import pandas as pd
+
+
+def load_data(file_path: str) -> pd.DataFrame:
+    """
+    Method used to load data.
+
+    Returns:
+        [pd.DataFrame]: Dataframe to be cleaned.
+    """
+    data_df = pd.read_csv(file_path, sep='\t')
+
+    return data_df
+
+def save_data(clean_df: pd.DataFrame, file_path: Path) -> None:
+    """
+    Method used to save data.
+    Args:
+        clean_df [pd.DataFrame]: Cleaned and filtered DataFrame to be saved.
```

Comment:
Missing file_path in docstring.

Reply:
Resolved in this commit edf699ceba8e788b4631b448506ff52dea924679